app/src/pages/CardSubmission/SubmissionForm.tsx

```diff
@@ -48,23 +50,32 @@ function SubmissionForm(props: { onSubmit: () => void }) {
     <>
       <Grid item>
         <Title
-          text={`Submit ${numToSubmit}
-          card${1 === numToSubmit ? "" : "s"}`}
+          text={t("title", "Submit {{ count }} card", {
+            count: numToSubmit,
+            defaultValue_plural: "Submit {{ count }} cards",
+          })}
         />
       </Grid>
 
       <Grid item>
-        These cards will be put into the "fishbowl," and drawn randomly in
-        rounds of Taboo, Charades, and Password. They can be words, familiar
-        phrases, or inside jokes!
+        {t(
+          "description",
+          'These cards will be put into the "fishbowl," and drawn randomly in rounds of {{ rounds }}. They can be words, familiar phrases, or inside jokes!',
+          {
+            rounds: currentGame.rounds
+              .map((round) => startCase(round.value))
+              .join(", "),
+          }
+        )}
       </Grid>
 
       {currentGame.starting_letter && (
         <Grid item>
-          <>
-            They must start with the letter{" "}
-            <b>{currentGame.starting_letter.toLocaleUpperCase()}</b>.
-          </>
+          <Trans t={t} i18nKey="descriptionLetter">
+            {"They must start with the letter "}
+            <b>{{ letter: currentGame.starting_letter.toLocaleUpperCase() }}</b>
+            .
+          </Trans>
```

Comment:
so we use components (Trans), when you want to interlace html in there?

Reply:
Updated in 957d64b761a54c0d1310f2dc6b308f416e6e1642.